src/include/executor/nodeShareInputScan.h

```diff
@@ -30,6 +30,8 @@
 #define NODESHAREINPUTSCAN_H
 
 #include "nodes/execnodes.h"
+
+#define MAX_RETRY_TIMES 5
```

Comment:
Is it appropriate to set the value as 5? What if the network environment is bad?